src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -642,7 +645,7 @@ function($tag) {
         }
     }
 
-    public static function provideFindByFiltersWithAudienceTargeting()
+    public static function provideFindByFiltersWithAudienceTargeting(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes